sw/otbn/code-snippets/p256.s

```diff
@@ -22,7 +22,7 @@ SetupP256PandMuLow:
   sw x2, 540(x0)
   addi x2, x0, 29
   bn.lid x2, 512(x0)
-  bn.wsrrw w31, 0, w29
+  bn.wsrw 0, w29
   bn.xor w31, w31, w31
```

Comment:
you can also remove the next line `bn.xor w31, w31, w31` then (was only there to clear the potentially clobbered w31 again)